src/devices/Gpio/Drivers/PinState.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Device.Gpio;
+
+namespace Iot.Device.Gpio.Drivers
+{
+    /// <summary>
+    /// Used to describe the pin status inside the driver.
+    /// </summary>
+    public class PinState
```

Comment:
the description sounds like this should be internal class yet it's public

Reply:
I created a field in `RockchipDriver`. I need to access this field in the subclass, so I used protected. If `PinState` is set to internal, there will be an access error. In fact, this class first appeared here, but there is no inheritance involved here https://github.com/dotnet/iot/blob/0a739b805b80b32e71cfb8875657d243b42174ba/src/System.Device.Gpio/System/Device/Gpio/Drivers/RaspberryPi3LinuxDriver.cs#L709
```C#
public unsafe class RockchipDriver : SysFsDriver
{
    protected IDictionary<int, PinState> _pinModes = new Dictionary<int, PinState>();
}
```